### PR TITLE
Stn tool

### PR DESCRIPTION
**Pull request by m312z**

### CODE REVIEW

File: rosplan_planning_system/src/STN_tool/replan_loop.py [hunk 1]
```diff
@@ -57,11 +60,10 @@
...
-    result.write('SUCCESS  '+ str(replans) + '\n')
-else:
-    print "FAILED ", str(replans)
-    result = open(problem_path[:-5] + '_result_batch' + str(experiment_batch) +'.txt', 'w')
-    result.write('FAILED  '+ str(replans) + '\n')
+file_path = "~/" + os.path.basename(problem_path)[:-5] + '_result_batch' +str(experiment_batch) +'.txt'
+elapsed = time.time() - snap
+elapsed_sim = rospy.get_time()
+success = "SUCCESS" if goal_achieved else "FAILED"
+with open(file_path, 'wt') as result:
```
Comment (mikand) [inline rosplan_planning_system/src/STN_tool/replan_loop.py]: @ParisaZehtabi @m312z
Here we need to use os.path.expanduser to make tilda work (on my cluster it does not work)
See: https://stackoverflow.com/a/30151425

Comment (m312z) [reply rosplan_planning_system/src/STN_tool/replan_loop.py]: Can you check the latest commit - is it correct now?

Comment (mikand) [reply rosplan_planning_system/src/STN_tool/replan_loop.py]: It should be perfect. I'll try now.

**Comment by ParisaZehtabi** [comment]
Final changes for running the experiments with Singularity

**Pull request merged**